modules/apps.nix

```diff
@@ -17,11 +17,6 @@ let
   getExe = x:
```

Comment:
lib.getExe is now in nixpkgs.

Reply:
the implementation is different, even though I do find it to be better, I want to leave it out of this PR since it is not strictly unused code

Reply:
Right.

Reply:
This `getExe` implementation is a bit picky. That might be a good thing, although it seems to challenge the design goal for flake-parts to be "unopinionated". The underlying goal is to make flake-parts suitable as a foundation for anyone anywhere.
That said, the _existence_ of this logic already makes the options more permissive than the flake schema, but I'd have to check the nix implementation to be sure.